src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Diagnostics/RazorDiagnosticsPublisher.cs

```diff
@@ -150,54 +150,54 @@ internal void ClearClosedDocuments()
         try
         {
             lock (PublishedRazorDiagnostics)
-            lock (PublishedCSharpDiagnostics)
-            {
-                ClearClosedDocumentsPublishedDiagnostics(PublishedRazorDiagnostics);
-                ClearClosedDocumentsPublishedDiagnostics(PublishedCSharpDiagnostics);
+                lock (PublishedCSharpDiagnostics)
```

Comment:
This one is a bit weird... this looks like a bug

Reply:
lol, didn't even see this because its whitespace only. May well just add braces I suppose?

Reply:
Let me see if the editor does the same thing on format. If it doesn't I'll just file a bug and manually fix.

Reply:
The editor does the same thing! Guess it's a Roslyn bug then 😅

Reply:
https://github.com/dotnet/roslyn/issues/70672